Makefile

```diff
@@ -51,8 +54,10 @@ test-simplecov:
 	docker build -f examples/simplecov/Dockerfile .
 
 publish:
-	$(AWS) s3 sync --acl public-read artifacts/bin s3://codeclimate/test-reporter
+	$(AWS) s3 cp --acl public-read --recursive artifacts/bin/ s3://codeclimate/test-reporter/ --exclude "*" --include "test-reporter-$(VERSION)*" --include "test-reporter-latest*"
```

Comment:
@pbrisbin 
WDYT about doing something like this? we can't really avoid overriding using a flag.
```
publish: VERSION_COUNT=$(shell $(AWS) s3 ls s3://codeclimate/test-reporter/test-reporter-$(VERSION) | wc -l)
publish:
	if [ $(VERSION_COUNT) -gt 0 ]; then \
		echo "Version already published, skipping publishing..."; \
	else \
		$(AWS) s3 cp --acl public-read --recursive artifacts/bin/ s3://codeclimate/test-reporter/ --exclude "*" --include "test-reporter-$(VERSION)*" --include "test-reporter-latest*"; \
	fi;
```